app/src/main/java/com/tvtracker/dto/FavoritedItem.kt

```diff
@@ -0,0 +1,12 @@
+package com.tvtracker.dto
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "favorited_table")
+class FavoritedItem (
+    @PrimaryKey val id: Int,
+    @ColumnInfo(name = "user") var name: String,
+    @ColumnInfo(name = "movieId") var movieId: Int
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.